Games/JV.AD/script.js

```diff
@@ -66,44 +60,66 @@ function generateRandom(min = 9, max = 15) {
 
   return rand;
 }
-//hypotenuse has to be greater than two other side 
-
- let s1 = (generateRandom()); 
- let s2 = Math.ceil(Math.random()*8);
- let s3 = Math.ceil(Math.random()*8);
-  
-  const hyp = new PIXI.Text(s1);
-  hyp.x = 130;
-  hyp.y = 120;
-  app.stage.addChild(hyp);
-  
-  const obj = new PIXI.Text(s2);
-  obj.x = 100;
-  obj.y = 260;
-  app.stage.addChild(obj);
-  
-  const adj = new PIXI.Text(s3);
-  adj.x = 5;
-  adj.y = 150;
-  app.stage.addChild(adj);
-  
-  const theta = new PIXI.Text('θ');
-  theta.x = 36;
-  theta.y = 72;
-  app.stage.addChild(theta);
-  
-  //insert trig ratio values in buttons 
-  btn1 = document.getElementById("btn1");
-  btn1.innerHTML = `${s2}/${s1}`;
-  
-  btn2 = document.getElementById("btn2");
-  btn2.innerHTML = `${s2}/${s3}`;
-  
-  btn3 = document.getElementById("btn3");
-  btn3.innerHTML = `${s3}/${s1}`;
-  }
+ 
+
+
+//creating function to be able to change trig ratios in question 
+function nextQuestion()
+{
+question = document.getElementById("question");
+question.innerHTML = `What is the trig ratio of ${trigratio}?`;
+
+ 
+//labelling sides of triangle
+s1 = (generateRandom()); //hypotenuse has to be greater than two other side 
+s2 = Math.ceil(Math.random()*8);
+s3 = Math.ceil(Math.random()*8);
+
+const hyp = new PIXI.Text(s1);
+hyp.x = 130;
+hyp.y = 120;
+app.stage.addChild(hyp);
+
+const obj = new PIXI.Text(s2);
+obj.x = 100;
+obj.y = 260;
+app.stage.addChild(obj);
+
+const adj = new PIXI.Text(s3);
+adj.x = 5;
+adj.y = 150;
+app.stage.addChild(adj);
+
+const theta = new PIXI.Text('θ');
+theta.x = 36;
+theta.y = 72;
+app.stage.addChild(theta);
+
+btn1.innerHTML = `${s2}/${s1}`;
```

Comment:
You have to determine which answer/option is the correct one and store that option, so that you can check whether or not the user got it right.